Source/Core/Core/PowerPC/Interpreter/Interpreter_Tables.cpp

```diff
@@ -147,14 +147,14 @@ static GekkoOPTemplate table19[] =
 {
 	{528, Interpreter::bcctrx,      {"bcctrx", OPTYPE_BRANCH, FL_ENDBLOCK, 1, 0, 0, 0}},
 	{16,  Interpreter::bclrx,       {"bclrx",  OPTYPE_BRANCH, FL_ENDBLOCK, 1, 0, 0, 0}},
-	{257, Interpreter::crand,       {"crand",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{129, Interpreter::crandc,      {"crandc", OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{289, Interpreter::creqv,       {"creqv",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{225, Interpreter::crnand,      {"crnand", OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{33,  Interpreter::crnor,       {"crnor",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{449, Interpreter::cror,        {"cror",   OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{417, Interpreter::crorc,       {"crorc",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
-	{193, Interpreter::crxor,       {"crxor",  OPTYPE_CR, FL_EVIL, 1, 0, 0, 0}},
+	{257, Interpreter::crand,       {"crand",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{129, Interpreter::crandc,      {"crandc", OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{289, Interpreter::creqv,       {"creqv",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{225, Interpreter::crnand,      {"crnand", OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{33,  Interpreter::crnor,       {"crnor",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{449, Interpreter::cror,        {"cror",   OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{417, Interpreter::crorc,       {"crorc",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
+	{193, Interpreter::crxor,       {"crxor",  OPTYPE_CR, FL_EVIL | FL_SET_CRn, 1, 0, 0, 0}},
```

Comment:
Yeah, I don't know about FL_EVIL, but that sounds reasonable. We are starting to run out of flags though; I guess we could raise the flags to 64-bit from 32-bit.